solutions/10.js

```diff
@@ -0,0 +1,13 @@
+const callFuncBig = (func,num1, num2,biggest, i=0)=>{
+  if(num1>num2){
+    biggest= num1;
+  }else{
+  biggest = num2;
+  }
+  if(i==biggest){
```

Comment:
Also, please check for these test cases: num1=0 and num2 =0 , num1 = -ve number and num2 =-ve number